### PR TITLE
Added missing ordering attribute

### PR DESCRIPTION
**Pull request by strayobject**
This is already implemented in more recent releases. However, the news app 4.3.2 does not work on 7.0.4 due to the lack of ordering attribute on the feed entity.  
Note that 4.3.2 is the last that oficially supports OCv7.

----

This fixes "BadFunctionCallException: ordering is not a valid attribute" error, which is rendering app v4.3.2 useless for oc7.04

### CODE REVIEW

**Comment by Raydiation** [comment]
The ordering attribute exists in master, see https://github.com/owncloud/news/blob/master/db/feed.php#L71

It is also defined in the database schema: https://github.com/owncloud/news/blob/master/appinfo/database.xml#L164

The version you are targeting is 4.3.2. This version does not define an ordering attribute in the database schema https://github.com/owncloud/news/blob/367a4eed30e9fa77a2aa5d7924f041e967f8567d/appinfo/database.xml

I can only imagine that you tried to updated to 5.x and although the app was not running on 7 it core migrated your database tables (dont know why) and added the ordering field. So it is safe to drop that column in your database and then you can remove your fix ;)

**Comment by Raydiation** [comment]
TL;DR: column is not in 4.3.2, looks like db was migrated, you can drop that column from the database

**Comment by strayobject** [comment]
Ah, I did not check the db, indeed that was the case. I have tried a fresh install of the newest version of News, only to learn that it was >= v8. Removed the folder and added the 4.3.2. I'm guessing db got installed before app was version checked.
Thanks for the info :)
